client/grpc_query_test.go

```diff
@@ -66,6 +66,22 @@ func (s *IntegrationTestSuite) TestGRPCQuery_TestService() {
 	s.Require().Equal("hello", testRes.Message)
 }
 
+func (s *IntegrationTestSuite) TestGRPCConcurrency() {
+	val0 := s.network.Validators[0]
+	clientCtx := val0.ClientCtx
+	clientCtx.GRPCConcurrency = true
```

Comment:
@czarcas7ic why not table-driven?

Reply:
Isn't there only two tests to do? Echo with flag enabled and echo without?

Reply:
yeah we could have 2 cases and minimize code duplication by avoiding the repetition of these lines:
```
	in := &testdata.EchoRequest{Message: "hello"}
	out := &testdata.EchoResponse{}
	err := clientCtx.Invoke(context.Background(), "/testdata.Query/Echo", in, out)
	s.Require().NoError(err)
	s.Require().Equal("hello", out.Message)
```

It's okay though, let's keep it. I just had the impression you wanted to make these table-driven when we talked